admin-console/src/main/webapp/src/pages/EmailTemplatePage.tsx

```diff
@@ -70,29 +70,34 @@ export const EmailTemplatePage: React.FunctionComponent = () => {
     }, []);
 
     const newTemplate = useCreateTemplate();
-    const [ templates, setTemplates ] = React.useState<Partial<Template>>();
+    const [ template, setTemplate ] = React.useState<Partial<Template>>({
+        data: defaultContentTemplate
+    });
 
     const handleChange = (value: string, event: React.FormEvent<HTMLInputElement>) => {
         const target = event.target as HTMLInputElement;
-        setTemplates(prev => ({ ...prev, [target.name]: target.value }));
+        setTemplate(prev => ({ ...prev, [target.name]: target.value }));
     };
 
     const handleCodeChange = (value: string) => {
-        setTemplates(prev => ({ ...prev, content: value }));
+        setTemplate(prev => ({ ...prev, data: value }));
     };
 
-    const handleSubmit = React.useCallback((template) => {
```

Comment:
This `template` variable was something else (i guess it could have been the event?).
Changed to use the `template` in the state